arboretum/technology/auditree/README.md

```diff
@@ -160,8 +193,38 @@ reported on.
    from auditree_central.provider.auditree.checks.test_compliance_config import ComplianceConfigCheck
    ```
 
+### Python Packages
+
+* Class: [PythonPackageCheck][check-python-packages]
+* Purpose: Compare the most recent Python package evidence with the evidence
+from the most recent historical version of that evidence found in the locker and
+checks that the `auditree-arboretum` and `auditree-framework` packages are at
+their most current release level.
+* Behavior: For every difference found between the two versions of evidence a
+warning is generated and reported on.  Warnings are also generated when the
+`auditree-arboretum` or `auditree-framework` packages being used are not at the
```

Comment:
Could other packages, picked up from config, be tested in this way? Thinking about things like my_compliance_checks

Reply:
I ultimately decided against making this configurable because we get into things like internal v external hosting of packages which could make things painful.  We could limit the configuration to packages on PyPI but that doesn't address packages hosted elsewhere.